### PR TITLE
Sprint1

### PR DESCRIPTION
**Pull request by hhthacker**
End of "Sprint 1"

### CODE REVIEW

File: Bangazon/Bangazon/DataAccess/CustomerDataAccess.cs.orig [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿using Bangazon.Models;
```
Comment (copypastedeveloper) [inline Bangazon/Bangazon/DataAccess/CustomerDataAccess.cs.orig]: This file needs to be deleted.

File: Bangazon/Bangazon/Web.config.orig [hunk 2]
```diff
@@ -0,0 +1,143 @@
+<<<<<<< HEAD
```
Comment (copypastedeveloper) [inline Bangazon/Bangazon/Web.config.orig]: This file needs to be deleted

File: Bangazon/Bangazon/packages.config.orig [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<<<<<<< HEAD
```
Comment (copypastedeveloper) [inline Bangazon/Bangazon/packages.config.orig]: this file needs to be deleted

**Comment by hhthacker** [comment]
deleted those files! merging.

**Pull request merged**